Targeted keywords: HTML/CSS/JS (#763)
Codecademy/docs: content/javascript/concepts/arrays/terms/shift/shift.md

  'Saturday',
  'Sunday',
];

const removed = daysOfWeek.shift();

console.log('Removed day:', removed);

console.log('Rest of week:', daysOfWeek);
```

This will produce the following output:

```shell
Removed day: Monday
Rest of week: [ 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday' ]
```

## Codebyte Example

> The steps in an `alfredoRecipe` array are followed by using the `.shift()` method in a `while` [loop](https://www.codecademy.com/resources/docs/javascript/loops) and logging to the console:

Reviewer: ```suggestion
The instructions of the `alfredoRecipe` array are followed by a `while` [loop](https://www.codecademy.com/resources/docs/javascript/loops) that prints each step of the recipe to the console:
```

Reviewer: The original felt a bit indirect
